debian/changelog

```diff
@@ -1,8 +1,16 @@
 ubuntu-image (3.10.3) UNRELEASED; urgency=medium
 
+  [Maciej Borzecki]
   * Pass snapdusergo build tag.
 
- -- Maciej Borzecki <maciej.borzecki@canonical.com>  Thu, 27 Nov 2025 12:48:12 +0000
+  [Alexis Cellier]
+  * i/statemachine: Use dpkg-divert instead of backup/restore
+  * i/statemachine: divert* functions return 2 functions instead of commands
+  * i/statemachine: factorize diversion of policy-rc.d
+  * i/statemachine: do not export Divert* functions
+  * internal: DpkgDivert move from statemachine to helper
```

Comment:
This may be a bit too detailed(?). Maybe most of it could be regrouped under "Refactor diversion to prepare chroot" (or something better).